frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ImportCluster/ImportCluster.tsx

```diff
@@ -20,58 +21,148 @@ import {
+    const [drawerMaxSize, setDrawerMaxSize] = useState<string | undefined>('1400px')
+
+    useResizeObserver(pageRef, (entry) => {
+        const inline = entry.contentRect.width > minWizardSize + defaultPanelSize
+        setDrawerInline(inline)
+        setDrawerMaxSize(inline ? `${Math.round((entry.contentRect.width * 2) / 3)}px` : undefined)
+    })
+
+    const [importResources, setImportResources] = useState<any | undefined>([])
+    function onFormChange(resources: any) {
+        setImportResources(resources)
+    }
+    const [editorChanges, setEditorChanges] = useState<SyncDiffType>()
+
     return (
-        <AcmPage
-            header={
-                <AcmPageHeader
-                    title={t('page.header.import-cluster')}
-                    breadcrumb={[
-                        { text: t('clusters'), to: NavigationPath.clusters },
-                        { text: t('page.header.import-cluster'), to: '' },
-                    ]}
-                    titleTooltip={
-                        <>
-                            {t('page.header.import-cluster.tooltip')}
-                            <a
-                                href={DOC_LINKS.IMPORT_CLUSTER}
-                                target="_blank"
-                                rel="noreferrer"
-                                style={{ display: 'block', marginTop: '4px' }}
+        <div ref={pageRef} style={{ height: '100%' }}>
+            <Page
+                additionalGroupedContent={
+                    <Fragment>
+                        <AcmPageHeader
+                            title={t('page.header.import-cluster')}
+                            breadcrumb={[
+                                { text: t('clusters'), to: NavigationPath.clusters },
+                                { text: t('page.header.import-cluster'), to: '' },
+                            ]}
+                            titleTooltip={
+                                <>
+                                    {t('page.header.import-cluster.tooltip')}
+                                    <a
+                                        href={DOC_LINKS.IMPORT_CLUSTER}
+                                        target="_blank"
+                                        rel="noreferrer"
+                                        style={{ display: 'block', marginTop: '4px' }}
+                                    >
+                                        {t('common:learn.more')}
+                                    </a>
+                                </>
+                            }
+                            switches={
+                                <Fragment>
+                                    <Switch
+                                        label="YAML"
+                                        isChecked={drawerExpanded}
+                                        onChange={() => {
+                                            localStorage.setItem('yaml', (!drawerExpanded).toString())
```

Comment:
The key for local storage `yaml` is very generic, like we would have 1 user setting for all wizards. However, currently the create cluster page is not using the same key. Do you think we should have 1 setting for everywhere where YAML is available, or a different one for each page?

Reply:
I'll do one for here for now--James' wizard will be the final arbitrator